app/services/import_ndc_sdg_targets.rb

```diff
@@ -0,0 +1,76 @@
+require 'csv'
+
+class ImportNdcSdgTargets
+  def call
+    cleanup
+    import_ndc_sdg_targets(read_from_s3('data/ndc_sdg_targets.csv'))
+  end
+
+  private
+
+  def cleanup
+    NdcSdg::NdcTargetSector.delete_all
+    NdcSdg::NdcTarget.delete_all
+    NdcSdg::Sector.delete_all
+  end
+
+  def read_from_s3(file_name)
```

Comment:
After merging the changes in my current branch we can reuse the S3CSVReader module I moved this function into

Reply:
We can do this now!

Reply:
hm it might need a slightly bigger refactor since the module already returns a `CSV` object. we can always do it later on.